cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaBridgeAssemblyOperatorTest.java

```diff
@@ -351,8 +340,15 @@ public void testCreateOrUpdateUpdatesCluster(VertxTestContext context) {
                 List<Deployment> capturedDc = dcCaptor.getAllValues();
                 assertThat(capturedDc, hasSize(1));
                 Deployment dc = capturedDc.get(0);
+                // getting the bridge ConfigMap to process the hash of the configuration to be checked on the corresponding Deployment annotation
+                List<ConfigMap> captureCm = cmCaptor.getAllValues();
+                assertThat(captureCm, hasSize(1));
+                ConfigMap cm = captureCm.get(0);
                 assertThat(dc.getMetadata().getName(), is(compareTo.getComponentName()));
-                assertThat(dc, is(compareTo.generateDeployment(Collections.singletonMap(Annotations.ANNO_STRIMZI_AUTH_HASH, "0"), true, null, null)));
+                assertThat(dc, is(compareTo.generateDeployment(Map.of(
+                        Annotations.ANNO_STRIMZI_AUTH_HASH, "0",
+                        KafkaBridgeCluster.ANNO_STRIMZI_IO_CONFIGURATION_HASH, Util.hashStub(cm.getData().get(KafkaBridgeCluster.BRIDGE_CONFIGURATION_FILENAME))
```

Comment:
Same as above. Does any of the test actually check that the value changes?

Reply:
I added a test for that.